common/app/model/Asset.scala

```diff
@@ -107,8 +107,9 @@ case class VideoAsset(
   // The video duration in seconds
   val duration: Int = fields.get("durationSeconds").getOrElse("0").toInt +
                            (fields.get("durationMinutes").getOrElse("0").toInt * 60)
+  val durationSeconds: Int = fields.get("durationSeconds").getOrElse("0").toInt
```

Comment:
We receive `durationMinutes: "0"` from the API - the reason for the change above is to not have the duration set to 0:00 when it isn't a media element.